src/NzbDrone.Api/Config/MediaManagementConfigModule.cs

```diff
@@ -1,16 +1,16 @@
 ﻿using FluentValidation;
 using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Validation;
 using NzbDrone.Core.Validation.Paths;
 
 namespace NzbDrone.Api.Config
 {
     public class MediaManagementConfigModule : NzbDroneConfigModule<MediaManagementConfigResource>
     {
-        public MediaManagementConfigModule(IConfigService configService, PathExistsValidator pathExistsValidator)
+        public MediaManagementConfigModule(IConfigService configService, PathExistsValidator pathExistsValidator, FileChmodValidator fileChmodValidator)
             : base(configService)
         {
-            SharedValidator.RuleFor(c => c.FileChmod).NotEmpty();
-            SharedValidator.RuleFor(c => c.FolderChmod).NotEmpty();
+            SharedValidator.RuleFor(c => c.FileChmod).SetValidator(fileChmodValidator).When(c => !string.IsNullOrEmpty(c.FileChmod));
```

Comment:
I was thinking about this and on windows the validator would still run and thus throw a NotSupportedException. probably better to add an OsInfo.IsMono in the When condition.